hail/src/main/scala/is/hail/io/bgen/LoadBgen.scala

```diff
@@ -47,10 +46,10 @@ case class BgenFileMetadata(
 )
 
 object LoadBgen {
-  def readSamples(hConf: org.apache.hadoop.conf.Configuration, file: String): Array[String] = {
-    val bState = readState(hConf, file)
+  def readSamples(fs: is.hail.io.fs.FS, file: String): Array[String] = {
```

Comment:
Kept in full form because I've encountered complaints about the import path not being resolvable in some recursive call cases. Because of this experience I became more conservative, and for first pass decided to keep fully qualified type specification if master did the same.